### DISCUSSION TITLE
How to config sink to Pulsar with tls key

### DISCUSSION THREAD

**Discussion by ngocchien** [vectordotdev/vector#16939] [Q&A] [answered]
Hi all,
I have read the document about the pulsar sink type. But I don't find TLS for config. 
Does anyone help me?  
Thank you so much.

Thanks,
ChienNguyen.

**Comment by jszwedko** [comment, marked as answer]
Unfortunately this is not yet supported: https://github.com/vectordotdev/vector/issues/10888

**Reply by ngocchien** [reply]
@jszwedko, thank you so much.
